fixes: ISSUE#1043 PLSQL create table AS should be followed by select statement instead of subquery

WolfeeTJ opened the pull request:
create table should be followed by select_statement instead of subquery
added drop table PURGE clause

KvanTTT commented on plsql/PlSqlParser.g4:
```diff
@@ -2078,7 +2078,7 @@ upgrade_table_clause
     
 drop_table
-    : DROP TABLE tableview_name SEMICOLON
+    : DROP TABLE tableview_name (PURGE)? SEMICOLON
```
Please remove redundant parenthesis.

WolfeeTJ commented:
@KvanTTT thank you for your review. I've removed the parenthesis and added an example. I'm new to contributing to grammars-v4 and not sure how unit test here works. So please let me know if the updates are good. Thank you.

KvanTTT commented:
Just add a code fragment which failed on the previous version of grammar to [examples](https://github.com/antlr/grammars-v4/tree/master/plsql/examples). Also, check other samples.
